Fraud-and-Money-Laundering-Detection-Fin-Services/db_scripts/queries/repeated_user.gsql

```diff
@@ -1,86 +1,94 @@
-CREATE QUERY repeated_user (VERTEX<User> receiver) for GRAPH AntiFraud syntax v2 {
- /*
-   Given a money receiver, this query is to discover whether there exists relationships among 
-   those people who have sent money to this receiver. 
+CREATE QUERY repeated_user (VERTEX<User> recipient)
+FOR GRAPH AntiFraud syntax v2 {
+/*
+   Given a money recipient, this query discovers whether there exists 
+   relationships among those people who have sent money to this receiver. 
    
-    1) Starting from a receiver "receiver", find all of her transactions receiving money.
+    1) Starting from a receiver "recipient", find all of her transactions
+    receiving money.
     2) Find all the senders from the transactions collected in step 1)
-    3) Start from the senders in step 2), go as far as 8 steps from each sender, 
-       find all the senders that are connected to other senders by a path made of 
-       Device_Token, Payment_Instrument, and Users.
-    4) Output all the transactions started by the senders found in step 3) and received by the input user.
+    3) Start from the senders in step 2), go as far as 8 steps from each 
+    sender, find all the senders that are connected to other senders 
+    by a path made of Device_Token, Payment_Instrument, and Users.
+    4) Detect all the transactions started by the senders found in step 3) 
+    and received by the input user.
 
     Sample input:
-    receiver: Recommend to use 1223 as input. Or, try integer between 1 and 500.
```

Comment:
@lunstad-tigergraph just out of curiosity, any reason why you just put this as 1223 but remove between 1 and 500 part?

Reply:
No idea, I believe this was a mistake